### ISSUE TITLE
Improve landing page

### ISSUE THREAD

**Issue by anikalindtner** [rails-girls-summer-of-code/rgsoc-teams#68]
for people who want to join as coaches or are looking for sth. or coaching companies...

**Comment by alicetragedy** [comment]
I know this is sort of old, but I like the idea of a landing page with a short blurb explaining what this is about. I'm happy to write the copy for it :)
And with @anikalindtner's permission, I'd like to change the title of the issue to that.. something like "improve landing page". Later on, if we feel there's a need for it, we could add that "Register" menu point.

**Comment by anikalindtner** [comment]
@alicetragedy DO EEEET <3

**Comment by sareg0** [comment]
I love the idea of improving the landing page. I know that it's not super fancy, but the rorganize.it app sort of tries to onboard people a bit. I'm not saying we have exactly the same thing, but just to give some inspiration.

**Comment by alicetragedy** [comment]
I know that we now have the "activities" as a landing page, which is quite nice — however in-between "seasons", we have an activity view of old teams .. so maybe a quick explanation of what the RGSoC Teams app is for might be better suited for that time of year when everything is quiet. Not sure how much work would go into that — but since I'm going through old issues, if this isn't a thing anymore and we're okay with keeping activities all year round, we should probably close it.

**Comment by carpodaster** [comment]
I kinda like the news stream, even if it's six months old now, but I agree that it may be odd to look at all the old activities. Otoh, people who visit the teams app probably know what they're looking at. In the unlikely event that they don't: wdyt about adding a box on top of the activity log giving a tiny bit of info what this is about but otherwise directing stranded internet citizens to the rgsoc.org homepage?

I'm also ok with just closing this.

**Comment by alicetragedy** [comment]
@carpodaster yeah, it's just that we had that conversation above apparently and I can see why there was the idea of adding a bit of information (a lot of people seem to be still confused about the use of the almighty monster teams app). Would love to take this on with, as you said, a box that gives some information about the app and redirects to rgsoc.org if needed.

**Comment by carpodaster** [comment]
@alicetragedy you assigned yourself to this issue … 1.5 years ago ;)  Is this still relevant for us?

**Comment by alicetragedy** [comment]
I think it's still relevant, because the news stream is a bit weird to see as first thing, when it's outside of the summer months. I could either un-assign myself and see if anyone is interested in giving it a shot, or we could close this issue entirely and see that more critical things are dealt with first, and revisit later?

**Comment by klappradla** [comment]
I'd keep it.
The "box on top of the activity stream" for instance that Carsten mentioned would be relatively fast and easy to add.
